src/classes/CRLP_RollupProcessor.cls

```diff
@@ -518,15 +511,10 @@ public inherited sharing class CRLP_RollupProcessor {
         // Determine if the updated SObject record is different than the parent.
         Boolean needsUpdate = CRLP_Rollup_SVC.resultsNeedUpdate(parent, updatedRecord, rollupDefs);
 
-        // Unconditionally set force skew flag if parent is Account/Contact and running in skew mode
-        if (mode == CRLP_RollupProcessingOptions.BatchJobMode.SkewMode && this.useForceSkewModeField && !isForceSkewModeTrue) {
-            updatedRecord.put(this.qualifiedSkewField, true);
-            needsUpdate = true;
-
-        // Reset any flags erroneously set for NonSkewMode records
-        } else if (mode == CRLP_RollupProcessingOptions.BatchJobMode.NonSkewMode && this.useForceSkewModeField && isForceSkewModeTrue) {
-            updatedRecord.put(this.qualifiedSkewField, false);
-            needsUpdate = true;
+        // Check whether skew mode threshold has been exceeded
+        initForceSkewModeValues(parent.getSObjectType());
+        if (mode == CRLP_RollupProcessingOptions.BatchJobMode.NonSkewMode && this.useForceSkewModeField) {
```

Comment:
The class variable useForceSkewModeField was introduced in response to feedback from @force2b to avoid making the call repeatedly since it will be the same for every call in a class instance.

Reply:
Yes. This method is called in a loop, so the goal was to only initialize the var a single time given that it's retrieving the sobject token for 2 object types. When rolling up opps for millions of contacts or account this can really add up. I'm not opposed to a different pattern either, but overall code should avoid repeating logic in a loop when it can be done a single time [for the batch scope].